pkg/cmd/release/config.go

```diff
@@ -0,0 +1,101 @@
+package release
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+	"github.com/openshift/ci-tools/pkg/registry"
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+type configOptions struct {
+	*options
+	list, resolve bool
+}
+
+func newConfigCommand(o *options) *cobra.Command {
+	co := configOptions{options: o}
+	ret := &cobra.Command{
+		Use:   "config",
+		Short: "ci-operator configuration commands",
+		Long: `Loads configuration files and writes them to stdout, optionally performing
+resolution.`,
+		RunE: func(_ *cobra.Command, args []string) error {
+			if 1 < util.PopCount(co.list, co.resolve) {
+				return fmt.Errorf("`list` and `resolve` are mutually exclusive")
+			}
+			if co.list {
+				return cmdConfigList(configPathsFromArgs(co.options, args))
+			}
+			if co.resolve {
+				if err := co.loadRegistry(); err != nil {
+					return err
+				}
+			}
+			if co.ciOperatorConfigPath != "" {
+				for i := range args {
+					args[i] = filepath.Join(co.ciOperatorConfigPath, args[i])
+				}
+			}
+			return cmdConfigPrint(co.resolver, configPathsFromArgs(co.options, args))
+		},
+	}
+	flags := ret.PersistentFlags()
+	flags.BoolVarP(&co.list, "list", "l", false, "list file names instead of contents")
+	flags.BoolVarP(&co.resolve, "resolve", "r", false, "resolve configurations before printing")
+	return ret
+}
+
+func configPathsFromArgs(o *options, args []string) []string {
+	if len(args) != 0 {
+		return args
+	}
+	if o.ciOperatorConfigPath != "" {
+		return []string{o.ciOperatorConfigPath}
+	}
+	return []string{filepath.Join(o.rootPath, config.CiopConfigInRepoPath)}
+}
+
+func cmdConfigList(paths []string) error {
```

Comment:
I didn't spend time on tests for this package since it's mostly connecting the existing functionality from other programs (which themselves are tested) to the command-line interface.

Other than arguably `registry.go`, I don't think there's much to be tested.